plugins/woocommerce-admin/client/marketplace/components/search/search.tsx

```diff
@@ -2,26 +2,15 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Icon, search } from '@wordpress/icons';
-import { useContext, useEffect, useState } from '@wordpress/element';
+import { useEffect, useState } from '@wordpress/element';
 import { navigateTo, getNewPath, useQuery } from '@woocommerce/navigation';
+import { SearchControl } from '@wordpress/components';
```

Comment:
I'm getting an error, `Module '"@wordpress/components"' has no exported member 'SearchControl'.` on this import, but it seems only the types aren't being exported. The component itself is being exported from `@wordpress/components`

Reply:
Yeah my build doesn't work because of it now 🤔

Reply:
As discussed on Slack, let's add ts-ignore for this line as otherwise the build won't work for many people. The problem is clearly with types for WP components

Reply:
Done in 042ac5d